stdlib/public/core/GroupInfo.json

```diff
@@ -214,5 +214,8 @@
     "Comparable.swift",
     "Codable.swift",
     "MigrationSupport.swift"
+  ],
+  "Result": [
```

Comment:
Rather than making a category just for `Result`, why not make an "Errors" category, move ErrorType.swift and Result.swift into it, and move it above "Misc"?

Reply:
I'll do whatever the team wants here. I just added a separate `Result` category because `Optional` had its own category.